aspnetcore/mvc/views/tag-helpers/intro.md

```diff
@@ -109,7 +109,11 @@ You must apply the Tag Helper opt-out character to the opening and closing tag.
 
 ### Using `@tagHelperPrefix` to make Tag Helper usage explicit
 
-The `@tagHelperPrefix` directive allows you to specify a tag prefix string to enable Tag Helper support and to make Tag Helper usage explicit. In the code image below, the Tag Helper prefix is set to `th:`, so only those elements using the prefix `th:` support Tag Helpers (Tag Helper-enabled elements have a distinctive font). The `<label>` and `<input>` elements have the Tag Helper prefix and are Tag Helper-enabled, while the `<span>` element does not.
+The `@tagHelperPrefix` directive allows you to specify a tag prefix string to enable Tag Helper support and to make Tag Helper usage explicit. For example, you could add the following markup to the *Views_ViewImports.cshtml* file:
```

Comment:
Missing a forward slash in the file name. It should be Views/_ViewImports.cshtml